Trisha-tech/OnlineBookSales
# Update hover effect

### Description
A clear and concise description of what the PR does.
- This PR does the following:
  - Updates hover effect on Book categories

### Related Issues
Link any related issues using the format `Fixes #issue_number`. 
This helps to automatically close related issues when the PR is merged.
- Placeholder: "Fixes #153 "

### Changes
List the detailed changes made in this PR.
- Added hover effect book categories section

### Testing Instructions
Detailed instructions on how to test the changes. Include any setup needed and specific test cases.
1. Pull this branch.
2. Run `npm install` to install dependencies.
3. Run `npm test` to execute the test suite.
4. Verify that .

### Screenshots (if applicable)
![Screenshot 2024-06-12 105642](https://github.com/Trisha-tech/OnlineBookSales/assets/128075021/3a07fe3c-90ea-46eb-84b4-42400c28a480)


### Additional Context
Any additional context or information that reviewers should be aware of.
- This PR is based on the following...

### Checklist
Make sure to check off all the items before submitting. Mark with [x] if done.
